build.gradle

```diff
@@ -19,6 +19,7 @@ repositories {
 dependencies {
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.10.1'
```

Comment:
We could update Jackson to 2.14.1

Maybe we take the opportunity to update other libs too, like Paho for example - it is really old.

Reply:
Sure, will take time to test and update all the dependencies